lib/types/project.dart

```diff
@@ -4,13 +4,14 @@ part 'project.g.dart';
 
 @JsonSerializable()
 class Project {
-  Project(this.id, this.name, this.slug);
+  Project(this.id, this.name, this.slug, this.platforms);
```

Comment:
a good example to be named params as a few of them have the same data type.
use `@required` if necessary

Reply:
That makes sense with many parameters. Currently, this constructor is only used in the generated code for json serialization. At least for now, i would leave it as is.

Reply:
true, if its annotation based, makes sense, missed the `JsonSerializable` thingie